brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Removed jQuery dependency from code.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/f42cc465-da50-48d9-b7e6-304b6609a533?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Improvement

### How has this been tested?

- Drag and drop Inline Notice and Notice Display > Dismisiable > enable cookies option and check if it is working on the frontend properly or not. Setting up cookies Notice should be disabled until cookies exit. Deleting cookies Notice should be visible.
- Drag and drop TOC, Add headings, and check if it is scrollable or not on the front. Enable Scroll to Top option and check if it is working properly or not.
- Drag and drop Tabs, Add content in every tab. & check tabs functionality.
- Drag and drop Lottie, Add a Lottie animation, Check Hover, click and Viewport event working fine or not.
- Drag and drop Forms, and check its functionality.
- Drag and drop Timeline, and check its functionality.
- Drag and drop the Carousel and Check if it is working as expected or not. Check some settings like Pause on hover, and Auto play.
- Drag and drop the Masonry and check load more/scroll pagination. Masonry functionality it should be work fine.
- Drag and drop Grid Add pagination and check if it is working fine or not.

### Checklist:
